File: problem01.py
arr = [10,15,3,7]
k = int(input())
flg = 1

#normal method
for i in range(len(arr)):
    temp = int(arr[i])
    for j in range(i+1,len(arr)):
        t = temp + int(arr[j])
        if(t == k):
            print(temp," + ",arr[j])
            flg = 0
            break
if(flg):
    print("null")

File: problem01.cpp
#include<stdio.h>
#include<conio.h>

int main()
{
	int arr[] = {10,7,15,3};
	int k,temp,i,j,t,flg=1;
	
	printf("enter the number\n");
	scanf("%d",&k);
	
	for(i=0;i<4;i++)
	{
		temp=arr[i];
		for(j=i+1;j<4;j++)
		{
			t=temp+arr[j];
			if(t==k)
			{
				printf("%d + %d",temp,arr[j]);
				flg=0;
				break;
			}
		}
	}
	
	if(flg)
	{
		printf("null");
	}
}
